services/src/main/java/org/keycloak/services/managers/ApplianceBootstrap.java

```diff
@@ -95,9 +95,15 @@ public boolean createMasterRealm() {
     public void createMasterRealmUser(String username, String password) {
         RealmModel realm = session.realms().getRealmByName(Config.getAdminRealm());
         session.getContext().setRealm(realm);
-
-        if (session.users().getUsersCount(realm) > 0) {
```

Comment:
My PR was indented for issue #16961, which was closed as a duplicate of #15789 at the time the PR was created.  Issue #16961 has since been reopened and I will move my PR to that issue.  ENV vars should create an admin specified even if another admin exists.  It is the only way to recover if new admin deletes the initial admin account/revokes the initial user's admin role or if the initial admin username/password is somehow lost.   In older versions of Keycloak, the ENV would create a new admin user regardless of the existence of other admin users.   This was extremely helpful because it also made Keycloak recovery from backup database possible.  If a Keycloak instance is wiped out, but a database backup exists, you can recover without knowing initial admin credentials, because a new admin account could be created with ENVs.  I strongly believe there needs to be a way to create an admin account in the event that someone loses initial admin password.